chef.gemspec

```diff
@@ -49,7 +49,8 @@ Gem::Specification.new do |s|
   s.add_dependency "net-ftp" # remote_file resource
   s.add_dependency "erubis", "~> 2.7" # template resource / cookbook syntax check
   s.add_dependency "diff-lcs", ">= 1.2.4", "!= 1.4.0", "< 1.6.0" # 1.4 breaks output. Used in lib/chef/util/diff
-  s.add_dependency "ffi-libarchive", "~> 1.0", ">= 1.0.3" # archive_file resource
+  # s.add_dependency "ffi-libarchive", "~> 1.0", ">= 1.0.3" # archive_file resource
+  s.add_dependency "ffi-libarchive", "~> 1.1", ">= 1.1.3"
```

Comment:
Go ahead and remove the commented out line and move the comment on it to the new line?

Reply:
Thanks, will do. I have a habit of leaving the old code in place if I am not sure the new thing is going to work.